### ISSUE TITLE
mst peerDep is outdated

### ISSUE THREAD

**Issue by barbalex** [agilgur5/mst-persist#32]
When running `npm i` this error occurs:
```
PS C:\Users\alexa\vermehrung> npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: vermehrung.ch@1.14.10
npm ERR! Found: mobx-state-tree@5.1.8
npm ERR! node_modules/mobx-state-tree
npm ERR!   mobx-state-tree@"5.1.8" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer mobx-state-tree@"^3.2.1" from mst-persist@0.1.3
npm ERR! node_modules/mst-persist
npm ERR!   mst-persist@"0.1.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! C:\Users\alexa\AppData\Local\npm-cache\_logs\2023-07-12T13_22_49_699Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\alexa\AppData\Local\npm-cache\_logs\2023-07-12T13_22_49_699Z-debug-0.log
```

This forces to install using `npm i --force` which is not recommended.

As no one will be using mobx-state-tree@^3.2.1 any more, it would help to update the peer dependency to a newer version to `Fix the upstream dependency conflict`.

**Comment by agilgur5** [comment]
Oh for sure, sorry I haven't kept this project updated in a bit

**Comment by barbalex** [comment]
@agilgur5  Thanks a lot for sharing this code, please don't excuse yourself for not being a super hero

**Comment by agilgur5** [comment]
Released in [`0.1.4`](https://github.com/agilgur5/mst-persist/releases/tag/v0.1.4)
